variables.tf

```diff
@@ -28,7 +28,7 @@ variable "mysql_version" {
 
 variable "region" {
   type        = string
-  description = "The region where you want to deploy your instance."
+  description = "The region to provision all resources in. [Learn more](https://terraform-ibm-modules.github.io/documentation/#/region) about how to select different regions for different services."
   default     = "us-south"
 }
 
```

Comment:
`memory_mb` input, in desc `per-member` should be without `-`, as `per member`
<img width="1146" alt="image" src="https://github.com/user-attachments/assets/ae126446-7a1f-4f48-82d1-b3cf36ae08cb" />

`auto_scaling` input, change the link in desc to point mysql autoscaling info